### PR TITLE
i18n(it): Updates `reference/frontmatter.md`

### PR DESCRIPTION
**Pull request by casungo**
#### Description

Updates `reference/frontmatter.md`

### CODE REVIEW

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). This looks great to me, I only have a few suggestions:

- I suggested a change to fix the link issue preventing the build to pass.
- It looks like [this change](https://github.com/withastro/starlight/pull/1530/files#diff-6d478912950527786985d16b5fef998fc9a98ffc282f20ebe557b822d0e717bd) is missing.

**Comment by delucis** [comment]
> I suggested a change to fix the link issue preventing the build to pass.

Looks like this suggestion went missing? I don’t see it in this PR.

File: docs/src/content/docs/it/reference/frontmatter.md [hunk 1]
```diff
@@ -268,6 +268,21 @@ pagefind: false
 ---
 ```
 
+### `draft`
+
+**tipo:** `boolean`  
+**predefinito:** `false`
+
+Imposta se questa pagina deve essere considerata una bozza e non essere inclusa nelle [build di produzione](https://docs.astro.build/en/reference/cli-reference/#astro-build) e nei [gruppi di link autogenerati](/guides/sidebar/#autogenerated-groups). Imposta su `true` per contrassegnare una pagina come bozza e renderla visibile solo durante lo sviluppo.
```
Comment (delucis) [inline docs/src/content/docs/it/reference/frontmatter.md:276]: This link needs fixing here to point to the Italian translation:

```suggestion
Imposta se questa pagina deve essere considerata una bozza e non essere inclusa nelle [build di produzione](https://docs.astro.build/en/reference/cli-reference/#astro-build) e nei [gruppi di link autogenerati](/it/guides/sidebar/#gruppi-generati-automaticamente). Imposta su `true` per contrassegnare una pagina come bozza e renderla visibile solo durante lo sviluppo.
```

**Review by delucis — COMMENTED** [review]
As @HiDeoo mentioned, there is still this one small update needed: https://github.com/withastro/starlight/pull/1530/files#diff-6d478912950527786985d16b5fef998fc9a98ffc282f20ebe557b822d0e717bd

**Review by delucis — APPROVED** [review]
Thanks for the update @casungo!

**Pull request merged**